dataflow/flex-templates/pipeline_with_dependencies/my_package/my_transforms.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Defines custom PTransforms and DoFns used in the pipleines."""
+
+import re
+from typing import Iterable
```

Comment:
As per [PEP 585](https://peps.python.org/pep-0585) this should be `from collections.abc import Iterable`. However, I remember Beam not playing nice with it a while ago, so you might need to add a `from __future__ import annotations` at the top and that should make it work.

Reply:
Done. Didn't have to import annotations.